storage/innobase/row/row0ins.cc

```diff
@@ -2711,8 +2727,8 @@ row_ins_clust_index_entry_low(
 	    && !trx->dict_operation
 	    && block->page.id().page_no() == index->page
 	    && !index->table->is_native_online_ddl()
-	    && (!dict_table_is_partition(index->table)
-	        || thd_sql_command(trx->mysql_thd) == SQLCOM_INSERT)) {
+            && (!dict_table_is_partition(index->table)
+                || thd_sql_is_insert(trx->mysql_thd))) {
```

Comment:
It seems that some TAB was inadvertently replaced with spaces in the indentation.